zer0-os/zOS
# feat: display group information panel when clicking group info menu item

### What does this do?
- adds stage to view group information panel when clicking the group info menu item in the group management menu dropdown.

### Why are we making this change?
- so a user is able to click the menu item and view the group information.

### How do I test this?
- open messenger > in the console `window.FEATURE_FLAGS.enableGroupInformation = true;` > navigate to a group conversation > click three dot group management menu trigger > select `Group Info` item > check group info panel is displayed in the sidekick.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
  
  

https://github.com/zer0-os/zOS/assets/39112648/aca3913c-837b-4fa4-8fa5-0b2abef389b5
